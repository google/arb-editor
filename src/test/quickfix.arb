{
	"@@locale": "en",
	"helloAndWelcome2": "Welcome!"
}