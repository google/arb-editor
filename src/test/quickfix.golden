{
	"@@locale": "en",
	"helloAndWelcome2": "Welcome!",
	"@helloAndWelcome2": {}
}